pramen/core/src/main/scala/za/co/absa/pramen/core/pipeline/OperationSplitter.scala

```diff
@@ -158,6 +158,8 @@ object OperationSplitter {
   val NOTIFICATION_TARGET_KEY = "notification.target"
   val NOTIFICATION_KEY = "notification"
 
+  private val logger = LoggerFactory.getLogger(this.getClass)
```

Comment:
```suggestion
  private val log = LoggerFactory.getLogger(this.getClass)
```
In the most of the project `log` is used as the name. But I'm fine with `logger` as well, up to you.

Reply:
Thanks for spotting this! I think I was working in Python too much and I forgot the convention here 😄 corrected :)